cmd/nerdctl/container_create.go

```diff
@@ -50,36 +52,530 @@ func newCreateCommand() *cobra.Command {
 	return createCommand
 }
 
-func createAction(cmd *cobra.Command, args []string) error {
+func processContainerCreateOptions(cmd *cobra.Command) (types.ContainerCreateOptions, error) {
```

Comment:
Suggest to use named return here (eg "opt types.ContainerCreatOptions, err error"), so you can simply "return" (instead of "return types.containercreateoptions{}, err"). You can take a look at "processContainerInspectOptions" for reference.